src/pages/EmergencyNumbers.jsx

```diff
@@ -0,0 +1,121 @@
+import { PhoneIcon } from '@chakra-ui/icons';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'PhoneIcon' is defined but never used.